src/qcodes/parameters/infer.py

```diff
@@ -0,0 +1,144 @@
+from __future__ import annotations
+
+from collections.abc import Sequence
+from typing import TYPE_CHECKING, ClassVar
+
+from qcodes.instrument import Instrument, InstrumentBase, InstrumentModule
+from qcodes.instrument.parameter import DelegateParameter, Parameter
```

Comment:
Parameters should be imported from qcodes.parameters. This module is a deprecated alias.

Reply:
Ooops. That's what I get for using Ruff's auto-import.